rest-client/src/main/scala/com/dota/rest/client/DataHandler.scala

```diff
@@ -1,41 +1,70 @@
 package com.dota.rest.client
 
-import org.mongodb.scala.{Completed, _}
+import java.util.concurrent.TimeUnit
 
-class DataHandler {
+import com.dota.rest.entity.Player
+import com.dota.rest.entity.matches.Match
+import org.mongodb.scala._
+import org.mongodb.scala.bson.collection.immutable.Document
+import org.mongodb.scala.model.Filters._
+
+import scala.concurrent.Await
+import scala.concurrent.duration.Duration
 
+class DataHandler {
+//ask about constructor params that are accesible from object
 }
 
 object DataHandler {
-  def put(id: Int, value:String): Unit = {
-
-    val client: MongoClient = MongoClient("mongodb://admins-MacBook-Pro.local:27017/") //the acrtual host name of parent machine
-
-    val listNamesObservable: Observable[String] = client.listDatabaseNames()
-
-    listNamesObservable.subscribe(new Observer[String] {
-
-      override def onError(e: Throwable): Unit = println(s"onError listNamesObservable: $e")
-      override def onComplete(): Unit = println("listNamesObservable onComplete")
-      override def onNext(result: String): Unit = println(s"onNext listNamesObservable: $result")
-    })
-
-    println("==========> BEFORE PUT")
-    println("==========> id")
-    println(id)
-    println("==========> value")
-    println(value)
-
-    val database = client.getDatabase("local")
-    val collection: MongoCollection[Document] = database.getCollection("test_collection")
-    Thread.sleep(5000)
-    val document: Document = Document("_id" -> id, "value" -> value)
-    val insertObservable: Observable[Completed] = collection.insertOne(document)
-    Thread.sleep(5000)
-    insertObservable.subscribe(new Observer[Completed] {
-      override def onNext(result: Completed): Unit = println(s"onNext insertObservable: $result")
-      override def onError(e: Throwable): Unit = println(s"onError insertObservable: $e")
-      override def onComplete(): Unit = println("insertObservable onComplete")
-    })
+  val client: MongoClient = MongoClient("mongodb://localhost:27017") //the actual host name of parent machine
```

Comment:
Maybe lets use incoming param for this ?
We can use default one for port for example